pkgs/development/tools/apksigcopier/default.nix

```diff
@@ -20,7 +19,7 @@ python3.pkgs.buildPythonApplication rec {
   };
 
   nativeBuildInputs = [ installShellFiles pandoc ];
-  propagatedBuildInputs = with python3.pkgs; [ click ];
+  propagatedBuildInputs = with python3.pkgs; [ apksigner click ];
```

Comment:
Apksigner is not a Python package, so it doesn't go in `propagatedBuildInputs`. Maybe you want to add it to `$PATH` using `makeWrapperArgs`?

Reply:
> Apksigner is not a Python package

True.

> so it doesn't go in propagatedBuildInputs.

Why not?

Reply:
> Why not?

Because adding the binaries of `propagatedBuildInputs` to `$PATH` was rather a coincidental development than intentional.

Reply:
So `makeWrapperArgs = ["--prefix" "PATH" ":" (lib.makeBinPath [ apksigner ])];`?

But would that still make the commands available to the test phase?

Reply:
> So `makeWrapperArgs = ["--prefix" "PATH" ":" (lib.makeBinPath [ apksigner ])];`?

Exactly.

> But would that still make the commands available to the test phase?

It wouldn't, you'll have to add the package to `checkInputs`.

Reply:
That seems a lot more complex and less intuitive to me than using `propagatedBuildInputs`. But sure. Thanks.